packages/api/src/db/subscription.ts

```diff
@@ -52,10 +52,20 @@ export enum SubscriptionSort {
 
 export interface SubscriptionFilter {
   readonly startsAt?: DateFilter
+  readonly startsAtFrom?: DateFilter
+  readonly startsAtTo?: DateFilter
   readonly paidUntil?: DateFilter
```

Comment:
Same question here.

Reply:
some of them are used in other methods, not related to the filtering itself so I decided to just leave them as they are (not wanting to break anything)

Reply:
Sounds reasonable